RELEASE_NOTES

```diff
@@ -1,3 +1,14 @@
+========================================================================
+Amazon CloudWatch Agent 1.300051.0 (2024-12-11)
+========================================================================
+Bug Fixes:
+* Fix Excessive IMDS related error logging
+* Set EKS.Cluster and K8s.Cluster without modifying global entity attribute map to prevent concurrent map writes
+
+Features:
+* Unify EMF export template in Application Signals to support adding RemoteEnvironment dimension for non-Kubernetes platforms.
```

Comment:
`[AppSignal] Support RemoteEnvironment dimension for non-Kubernetes platforms` or something like this is probably more clear.